deps/rabbit/src/rabbit_db.erl

```diff
@@ -100,15 +100,10 @@ init_using_mnesia() ->
     rabbit_sup:start_child(mnesia_sync).
 
 init_using_khepri() ->
-    case rabbit_khepri:members() of
-        [] ->
-            timer:sleep(1000),
-            init_using_khepri();
-        Members ->
-            ?LOG_WARNING(
-               "Found the following metadata store members: ~p", [Members],
-               #{domain => ?RMQLOG_DOMAIN_DB})
-    end.
+    ?LOG_DEBUG(
+      "DB: initialize Khepri",
+      #{domain => ?RMQLOG_DOMAIN_DB}),
+    rabbit_khepri:init().
```

Comment:
@dumbbell I have addressed one of your reviews https://github.com/rabbitmq/rabbitmq-server/pull/11879#discussion_r1713487430 in this PR instead since it doesn't depend on the new Khepri version and also needs to hook into Khepri's init

Reply:
Great!